src/Chapter2.hs

```diff
@@ -500,7 +515,9 @@ True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
+isThird42 :: (Num a, Eq a) => [a] -> Bool
+isThird42 (_ : _ : x : _) = x == 42
+isThird42 _               = False
```

Comment:
Great! You made it work with the wider number of types 👍🏼 

Also, note, that you could pattern match on the number as well, so you can write:

```haskell
isThird42 (_ : _ : 42 : _) = True
```